catchup/service.go

```diff
@@ -494,11 +495,26 @@ func (s *Service) pipelinedFetch(seedLookback uint64) {
 				return
 			}
 
+			// if ledger is busy, pause for some time to let the fetchAndWrite goroutines to finish fetching in-flight blocks.
+			start := time.Now()
```

Comment:
There is also a `start` variable at the beginning of `sync`, which is the only caller of `pipelinedFetch`. Would it make sense to use that instead, or does it not matter for some reason?

Reply:
it does not matter, the entire point of this to have bounded sleep